### ISSUE TITLE
Texture Scaling Alignment & Render Resolution

### ISSUE THREAD

**Issue by 7331Squall** [hrydgard/ppsspp#2901]
I am aware that you guys are aware of these issues, but since I'm guessing it happens with every game, it's no longer adequate to call it "Square Games issues" like I did in #2065. Which I just closed, since the main issues in it were fixed as of 0.8.1.749.

By the way, Congratulations on that! :)

Now, the only issues left are:
- Some Texture Alignment issues when the rendering resolution is not 1X and if Anti-Aliasing is off.

![ccissues](https://f.cloud.github.com/assets/4536138/844700/1b03c840-f3d8-11e2-9eec-d70d6b08ccee.jpg)
This shows up the issue very well. At 2xH you can see a little line near Infantryman, and at 3xB you can see several issues.
- Speaking of the Texture Scaling, it only works when I enable it from the dropdown menu. If I go to the "Emulation Paused > Settings > Graphics" it won't work.
- Render Resolution won't work when you fullscreen or when you resize the window. It'll always assume the Window Size as the Render Resolution.

The problem is, when you fullscreen, it'll assume your desktop size as the rendering size, and my PC ain't all that powerful to handle everything I turn on.

A solution I got for this one is Saving State, then quitting the game, then leaving fullscreen, then pressing Ctrl+1 (or whatever setting I want the emulator to keep) then entering fullscreen again (what makes the menu drastically reduce its button sizes) then clicking to reopen the game, then loading the saved state.

This needs to be done everytime I need to do something else, what includes Alt+Tabbing.

Those 3 issues are now the only ones I'm having.

**Comment by unknownbrackets** [comment]
I haven't checked this alignment specifically, but I think it's fixed now when texture scaling and render resolution are increased.  A lot of the mentioned settings don't exist exactly the same way anymore.

-[Unknown]

**Comment by hrydgard** [comment]
I don't think remaining problems caused by upscaling are solvable in any generic way anyway. Closing.

**Comment by 7331Squall** [comment]
I... dropped these games a good while back, when I accidentally deleted my whole Hard Drive's Partition, saves along with it and I managed to buy a PSP past year, so I kinda stopped keeping up with the development, as I've restarted them all on real hardware... XD

Still, I'm up to grabbing my CSO of Crisis Core from my PSP and downloading the latest version to really test this issue again. I really believe it is fixed, A LOT has changed.

But, just for peace of mind.... Wouldn't hurt anyone, right XD?
